test/CadlRanchProjects.Tests/property-optional.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Xml;
+using AutoRest.TestServer.Tests.Infrastructure;
+using Azure;
+using Models.Property.Optional;
+using NUnit.Framework;
+using property_optional;
```

Comment:
Is it possible to make the namespace and class name different?

Reply:
Yes, we could change the namespace of generated SDK to Property.Optional